angular-apollo-tailwind/src/app/profile/profile-about/profile-about.component.html

```diff
@@ -74,7 +74,7 @@ <h1 class="mt-2">
           ></hero-icon>
           <a
             class="link"
-            [href]="user.websiteUrl"
+            [href]="generateValidUrl(user.websiteUrl)"
```

Comment:
Could you please implement this as pipe? Calling functions in template is not good for performance

Reply:
oh sure.

Reply:
done @ktrz